src/Iterator/GlobIterator.php

```diff
@@ -33,9 +33,12 @@ class GlobIterator extends IteratorIterator
      *
      * @param string $glob  The glob pattern.
      * @param int    $flags A bitwise combination of the flag constants in
-     *                      {@link Glob}.
+     *               {@link Glob}.
+     * @param bool   $skip_errors Add the RecursiveIteratorIterator::CATCH_GET_CHILD
+     *               to internal RecursiveIteratorIterator and prevent throwing
+     *               Exception on errors like access denied.
      */
-    public function __construct($glob, $flags = 0)
+    public function __construct($glob, $flags = 0, $skip_errors = false)
```

Comment:
This library requires PHP 7.3 or newer, so we can safely use `bool` here

Reply:
Let's also call this `$skipErrors`

Reply:
Agree will push an update shortly.

Reply:
I also added types for the other two arguments.